sklearn/ensemble/_gb.py

```diff
@@ -434,15 +434,20 @@ def fit(self, X, y, sample_weight=None, monitor=None):
 
         if self.n_iter_no_change is not None:
             stratify = y if is_classifier(self) else None
-            X_train, X_val, y_train, y_val, sample_weight_train, sample_weight_val = (
-                train_test_split(
-                    X,
-                    y,
-                    sample_weight,
-                    random_state=self.random_state,
-                    test_size=self.validation_fraction,
-                    stratify=stratify,
-                )
+            (
+                X_train,
+                X_val,
+                y_train,
+                y_val,
+                sample_weight_train,
+                sample_weight_val,
+            ) = train_test_split(
+                X,
+                y,
+                sample_weight,
+                random_state=self.random_state,
+                test_size=self.validation_fraction,
+                stratify=stratify,
             )
             if is_classifier(self):
                 if self._n_classes != np.unique(y_train).shape[0]:
```

Comment:
The change in `sklearn/ensemble/_gb.py` is unrelated to the purpose of this PR. Can you please restore this file to it original state?